ydb/_topic_reader/topic_reader_asyncio_test.py

```diff
@@ -1127,6 +1127,32 @@ async def test_update_token(self, stream):
 
         await reader.close()
 
+    async def test_read_unknown_message(self, stream, stream_reader):
+        class TestMessage:
+            pass
+
+        logged = asyncio.Event()
+
+        def set_logged(*args, **kwargs):
+            logged.set()
+
+        stream_reader._logger = mock.Mock()
```

Comment:
I think there are more beautiful and simple way to check log in pytest.
looks at https://docs.pytest.org/en/7.1.x/how-to/logging.html#caplog-fixture